src/Models/UserModel.php

```diff
@@ -49,6 +49,20 @@ class UserModel extends Model
      */
     protected ?User $tempUser = null;
 
+    /**
+     * Auth Identities table name
+     */
+    private string $tableIdentities;
+
+    protected function initialize(): void
+    {
+        /** @var Auth $authConfig */
+        $authConfig = config('Auth');
+
+        $this->table           = $authConfig->tables['users'];
+        $this->tableIdentities = $authConfig->tables['identities'];
```

Comment:
I didn't like this, I prefer it as below to maintain the consistency of using table names throughout the code.
```php
$this->table           = $authConfig->tables['users'];
$this->tables['users'] = $authConfig->tables['users'];
$this->tables['identities'] = $authConfig->tables['identities'];
```

Reply:
Done.

Reply:
Thanks!